templates/zerver/help/mute-a-topic.md

```diff
@@ -38,6 +38,33 @@ Follow the same procedure to unmute the topic.
 
 Follow the same procedure to unmute the topic.
 
+### From the slashed bell icon
+
+{start_tabs}
+
+1. Go to the **Recent topics** or find the recipient bar of the topic that you wish
+   to mute.
+
+2. Hover over the slashed bell icon (<i class="fa fa-bell-slash"></i>)
+   and click on it.
+
+{end_tabs}
+
+Follow the same procedure to unmute the topic through recipient bar.
+
+Mark **Include muted** as checked and follow the same procedure to
+unmute the topic through **Recent topics**.
+
+### From the Settings
+
+{start_tabs}
+
+{settings_tab|muted-topics}
+
+1. Click **Unmute** to unmute the topic of your choice under the **Actions** column.
```

Comment:
`Click **Unmute** to unmute the topic that you wish to unmute under the ......` sounds a bit odd so I've added the above line.

Reply:
This section is basically a duplicate of the "See ..." one below; I think we should need a better heading, maybe "Browse and unmute muted topics"

Reply:
I've merged this section as a `tip` in the next section.